localstripe/resources.py

```diff
@@ -2304,16 +2363,7 @@ def _create_invoice(self):
             if ii.invoice is None]
 
         for si in self.items._list:
-            pending_items.append(
-                InvoiceItem(subscription=self.id, plan=si.plan.id,
-                            amount=si._calculate_amount(),
-                            currency=si.plan.currency,
-                            description=si.plan.name,
-                            tax_rates=[tr.id
-                                       for tr in (si.tax_rates or [])],
-                            customer=self.customer,
-                            period_start=self.current_period_start,
-                            period_end=self.current_period_end))
+            pending_items.append(si)
```

Comment:
The code looks good, but can it be a problem to reuse the same objects here (instead of creating new objects, like the previous code did)? @H--o-l I think you reworked this code not so long ago (commit 5fca63eb), do you have an opinion?

Reply:
Hey!

For me it looks OK, because a new object is not created here, but will be at this line https://github.com/adrienverge/localstripe/pull/119/files#diff-62ee820ebbbef6539c97fecb3a10c374R915

Object that are created and should not be removed without breaking something are :
https://github.com/adrienverge/localstripe/blob/c6821928953c50058327f04ef11b9e70bc8e609b/localstripe/resources.py#L2471-L2476
These end-ups being the one we get with:
```py
        pending_items = [ii for ii in InvoiceItem._api_list_all(
            None, customer=customer, limit=99)._list
            if ii.invoice is None]
```
TL;DR: OK for me! @baptsite as said on Slack, unit test could be cool, and the following API https://stripe.com/docs/api/invoices/invoice_lines too otherwise OK for me.

Reply:
@H--o-l thanks :+1: